duckduckgo/Android
# Bottom Navigation

Task/Issue URL: https://app.asana.com/0/488551667048375/1168086989855979
Pixels and Dashboard: https://app.asana.com/0/1157893581871903/1168245052250051 
Product Changes: https://app.asana.com/0/1157893581871903/1170128420609770

**Description**:
Device comfort on our Android app is pretty bad - on larger devices it’s impossible to use our app with one hand and we get consistent (though low to mid frequency) around this. 
Material design has bottom navigation options that we could leverage to improve device comfort and bring the navigation patterns closer to what we have on iOS, which is better when it comes to device comfort (with room to improve though).
I’d like to experiment around migrating the primary navigation (if not all of it) to the bottom of the screen to see what impact it has on retention.

**Steps to test this PR**:
**Concept test variant mf: Control group**
1. Open app
2. Navigate to a website
3. Check that bottom bar is not present

**Concept test variant mm: Bottom Navigation**
1.  Open app
2. Navigate to a website
3. Check that bottom bar is present

**Steps to test Pixels**

Each pixel event has a suffix that identifies either the control group or the experiment group.
- Control group: `mm`
- Experiment group: `mn`

1. Open Popup menu, pixel should fire `m_nav_pm_o_mm` or  `m_nav_pm_o_mn`
2. Press fire icon, pixel should fire `m_nav_f_p_mm` or  `m_nav_f_p_mn`
3. Press Refresh in the Popup Menu, pixel should fire `m_nav_r_p__mm` or  `m_nav_r_p_mn`
4. New Tab pressed, pixel should fire `m_nav_nt_p_mm` or  `m_nav_nt_p_mn`
5. Bookmarks button pressed, pixel should fire `m_nav_b_p_mm` or  `m_nav_b_p__mn`
6. Search Loupe pressed (only in the experiment group) pixel should fire `m_nav_s_p_mn`

**Screenshots**:

***Light theme***:
| Bottom bar | Omnibar focused | Popup Menu | Tabs |
| ------ | ----- | ----- | ----- | 
![device-2020-04-17-141414](https://user-images.githubusercontent.com/531613/79568640-7d21ed00-80b6-11ea-93bd-7169699c76e5.png)|![device-2020-04-17-141406](https://user-images.githubusercontent.com/531613/79568642-7dba8380-80b6-11ea-9da2-a971e78f622e.png)|![device-2020-04-17-141433](https://user-images.githubusercontent.com/531613/79568632-7a26fc80-80b6-11ea-991e-bd1101dfa5ca.png)|![device-2020-04-17-141422](https://user-images.githubusercontent.com/531613/79568638-7bf0c000-80b6-11ea-8887-ee40e27a6913.png)|

***Dark theme***:
| Bottom bar | Omnibar focused | Popup Menu | Tabs |
| ------ | ----- | ----- | ----- | 
![device-2020-04-17-141447](https://user-images.githubusercontent.com/531613/79568816-d9850c80-80b6-11ea-8049-d0f62da72772.png)|![device-2020-04-17-141456](https://user-images.githubusercontent.com/531613/79568812-d7bb4900-80b6-11ea-9beb-1216f715cb85.png)|![device-2020-04-17-141504](https://user-images.githubusercontent.com/531613/79568810-d68a1c00-80b6-11ea-924e-d615b736fd34.png)|![device-2020-04-17-141511](https://user-images.githubusercontent.com/531613/79568804-d4c05880-80b6-11ea-82f8-aa33d882378b.png)|
